### PR TITLE
enable auto-google-login on test devices

### PR DESCRIPTION
**Pull request by fadidurah**
Quick PR to add the auto-google-login arguement to broker CI pipeline, allowing runner to login to a google account on the physical testing device to access play store and other features.

### CODE REVIEW

File: azure-pipelines/ui-automation/broker-test.yml [hunk 1]
```diff
@@ -229,6 +229,7 @@ stages:
             --type instrumentation \
             --app "$(Pipeline.Workspace)/msalautomationapks/$(msalApp)" \
             --test "$(Pipeline.Workspace)/msalautomationapks/$(msaltestApp)" \
+            --auto-google-login \
```
Comment (shahzaibj) [inline azure-pipelines/ui-automation/broker-test.yml:232]: Wasn't this already happening? I thought this was a default?

Comment (fadidurah) [reply azure-pipelines/ui-automation/broker-test.yml:232]: Yes it is set to true by default when using gcloud, but flank actually sets it to false by default, same thing with record-video

**Review by shahzaibj — APPROVED** [review]
I'm approving this, but I hope we will leverage the template that you created earlier for these pipelines as well.

**Comment by fadidurah** [comment]
> I'm approving this, but I hope we will leverage the template that you created earlier for these pipelines as well.

I'm thinking I will implement the template structure sometime late september if my other items permit, It's not really blocking anything at the moment and I think my other items are higher priority.

**Pull request merged**